bower.json

```diff
@@ -46,8 +46,8 @@
     "angular-moment": "1.0.0",
     "angular-utf8-base64": "0.0.5",
     "file-saver": "1.3.3",
-    "origin-web-common": "0.0.52",
-    "origin-web-catalog": "0.0.42"
+    "origin-web-common": "0.0.53",
+    "origin-web-catalog": "0.0.43"
```

Comment:
#1987 bumps common and catalog (and is required due to API changes in common)

Reply:
Removing and updating.